packages/e2e/ios/Classes/E2EPlugin.m

```diff
@@ -27,9 +27,14 @@ - (instancetype)initForRegistration {
 }
 
 + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
+}
+
+- (void)setupChannels:(id<FlutterBinaryMessenger>)binaryMessenger {
```

Comment:
Maybe add a comment here for why you did this, mentioning the static issue if e2e plugin is linked as a static library.

Reply:
Done.